tardis/adapters/sites/slurm.py

```diff
@@ -127,5 +127,7 @@ def handle_exceptions(self):
             raise TardisResourceStatusUpdateFailed
         except TimeoutError as te:
             raise TardisTimeout from te
+        except KeyError as ke:
```

Comment:
Is this enough to solve the problem mentioned in https://github.com/tardis-resourcemanager/tardis/pull/46#discussion_r271192317 ? Maybe you can take a look how this is solved in the HTCondor site adapter https://github.com/tardis-resourcemanager/tardis/blob/7f2798fd3955e258eefa46cb80efd667a5c54aa7/tardis/adapters/sites/htcondor.py#L96.

Reply:
In case your resource vanishes from SLURM, you will try to update the resource status forever.